src/libs/tools/lighttable.c

```diff
@@ -323,7 +323,7 @@ static float _action_process_preview(gpointer target, dt_action_element_t elemen
 
 const dt_action_element_def_t _action_elements_preview[]
   = { { "normal", dt_action_effect_hold },
-      { "focus detection", dt_action_effect_hold },
+      { N_("focus detection"), dt_action_effect_hold },
```

Comment:
Why not the one above as well?

Reply:
For a moment I did not take it into account because the same form is in several languages. But I agree, especially for those who use another alphabet.